src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -44,6 +44,33 @@ def register_options(cls, register):
              help='Requirements string for the unittest2 library, which some python versions '
```

Comment:
Documentation will need to be updated from V1. The main thing to clarify is that timeouts are against the _target_, whereas now it uses phrases like `for a test`. For example, timeout default should be `'The default timeout (in seconds) for a test target if the timeout field is not set on the target.'`